logging/cluster-logging-dashboards.adoc

```diff
@@ -9,11 +9,11 @@ The *Logging/Elasticsearch Nodes* and *Openshift Logging* dashboards in the {pro
 
 The *OpenShift Logging* dashboard contains charts that show details about your Elasticsearch instance at a cluster level, including cluster resources, garbage collection, shards in the cluster, and Fluentd statistics.
 
-The *Logging/Elasticsearch Nodes* dashboard contains charts that show details about your Elasticsearch instance, many at node level, including details on indexing, shards, resources, and so forth. 
+The *Logging/Elasticsearch Nodes* dashboard contains charts that show details about your Elasticsearch instance, many at node level, including details on indexing, shards, resources, and so forth.
 
 [NOTE]
 ====
-For more detailed data, click the *Grafana UI* link in a dashboard to launch the Grafana dashboard. Grafana is shipped with xref:../monitoring/understanding-the-monitoring-stack.adoc#understanding-the-monitoring-stack[OpenShift cluster monitoring].
+For more detailed data, click the *Grafana UI* link in a dashboard to launch the Grafana dashboard. Grafana is shipped with xref:../monitoring/overview-of-monitoring.adoc#overview-of-monitoring[Overview of monitoring].
```

Comment:
Re-phrased as : Grafana is shipped with xref:../monitoring/overview-of-monitoring.adoc#overview-of-monitoring [OpenShift cluster monitoring] since the title is now `overview of monitoring`